docs/installation.rst

```diff
@@ -121,3 +121,71 @@ email address. Then we could do this:
 Then you'd use the Python dotted path to that class in the
 ``settings.AUTHENTICATION_BACKENDS`` instead of
 ``mozilla_django_oidc.auth.OIDCAuthenticationBackend``.
+
+
+Creating a Django ``User`` record for new users
+-----------------------------------------------
+
+If a user logs into your site and doesn't already have an account, by default,
```

Comment:
It's definitely worth mentioning `OIDC_CREATE_USER`, but I don't really understand it.

If you have it set to False, how does the flow work? Don't you end up with a user that's authenticated, but has no corresponding Django user? What do you do at that point?